packages/utils/src/types.ts

```diff
@@ -9,6 +9,7 @@ export type SerializedStyles = {
   styles: string
   map?: string
   next?: SerializedStyles
+  toString: () => string
```

Comment:
could you point me to a line that required this change?

Reply:
https://github.com/emotion-js/emotion/pull/2543/files#diff-ec2de171b3fd05a04be89b82f3a9f0beb0768838544583edc37162fcf9762c62R201

Reply:
maybe it would be enough to change that call-site to `String(interpolation)`?